integration/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+require_relative "supports/logstash_test_helpers"
```

Comment:
What is wrong with `supports?` Remove the s, rename it?

lets drop the logstash and called it: `integration_test_helpers`